main/lsp/LSPPreprocessor.cc

```diff
@@ -503,5 +506,16 @@ LSPPreprocessor::canonicalizeEdits(uint32_t v, unique_ptr<WatchmanQueryResponse>
     }
     return edit;
 }
+const std::unique_ptr<std::vector<std::string_view>> LSPPreprocessor::openFilePaths() const {
+    auto paths = make_unique<std::vector<std::string_view>>();
+    for (auto const &[path, file] : openFiles) {
+        paths->emplace_back(path);
+    }
+    return paths;
+}
```

Comment:
I have half a mind to suggest that we keep openFilePaths private and also call it in getTaskForMessage while constructing a DidChangeConfigurationTask, but I feel less strongly about that.

Reply:
just got rid of it